spring-boot/proxy/src/main/resources/application.yml

```diff
@@ -34,7 +34,7 @@ zuul:
 logging:
   level:
     eu.coatrack.proxy: DEBUG
-    org.springframework.security: DEBUG
+    org.springframework.security: INFO
```

Comment:
We should double-check that this change does not hide relevant information required for debugging.